### PR TITLE
docs, config: render logging related options

### PR DESCRIPTION
**Pull request by tchaikov**
this changeset adds a filter to customize the rendering of default values, and enables the `scylladb_cc_properties` extension to display the logging message related options. it prepares for the further improvements in https://opensource.docs.scylladb.com/master/reference/configuration-parameters.html.

this changeset also prepare for the improvements requested by #19463

---

it's an improvement in the document, hence no need to backport.

### CODE REVIEW

**Comment by tchaikov** [comment]
@dgarcia360 hi David, could you help review this change?

**Comment by tchaikov** [comment]
before this change:
---

https://opensource.docs.scylladb.com/master/reference/configuration-parameters.html#confval-enable_tablets

after this change
---

http://previews.docs.scylladb.com/scylladb/19483/reference/configuration-parameters/#confval-logger_log_level

**Comment by tchaikov** [comment]
@dgarcia360 hi David, could you please take a look?

File: docs/_ext/utils.py [hunk 1]
```diff
@@ -35,10 +35,41 @@ def readable_desc_rst(description):
...
+    # representation in source code of a certain value does not necessarily
+    # identical to its representation in scylla.yaml, sometimes the default
+    # value is not even a literal. strictly speaking, it is an expression,
+    # so a function call is also valid. we have to translate them on a
+    # case-by-case basis
+    if type_ == 'tri_mode_restriction':
+        if value == 'strict_allow_filtering_default()':
+            # see db/config.cc
+            #   strict_allow_filtering_default()
+            return "warn"
```
Comment (dgarcia360) [inline docs/_ext/utils.py:49]: I would refrain from hardcoding values, as they might not be updated here if the code changes. Instead, we could consider leaving references to the functions (cleaning them if necessary to make sure they're readable). 

This approach might slightly impact user experience but ensures accurate, up-to-date documentation without the need to update the script constantly.

Additionally, we could explore ways to enhance its usefulness as part of a separate PR, such as linking to the code or better resolving the function.

Comment (tchaikov) [reply docs/_ext/utils.py:49]: i understand the downside of this approach, but, IMHO, a regular user should not have the source code for using a software. it'd be difficult to address this without evaluating the function to get the value returned by it. anyway, i am extracting this commit into anther PR, and remove the part which is controversial.

Comment (tchaikov) [reply docs/_ext/utils.py]: moved this change to #19566

**Comment by tchaikov** [comment]
v2:

- extract the last commit into #19566

File: docs/_ext/utils.py (hunk 1 above)
Comment (tchaikov) [reply docs/_ext/utils.py]: > i understand the downside of this approach, but, IMHO, a regular user should not have to read the source code for using a software. it'd be difficult to address this without evaluating the function to get the value returned by it. anyway, i am extracting this commit into anther PR, and remove the part which is controversial.

another way forwards is to actually compile and run this part of code into a tiny program which dumps the values of the option in a form which can be used directly in `scylla.yaml`, and parse the output of this program instead.

**Review by dgarcia360 — APPROVED** [review]
LGTM

**Comment by tchaikov** [comment]
@scylladb/scylla-maint hello maintainers, could you help review this change as well?

**Pull request closed without merging**